src/Command/Project/ProjectCreateCommand.php

```diff
@@ -216,6 +273,39 @@ protected function getAvailableRegions($runtime = false)
         return $regions;
     }
 
+    /**
+     * Return the catalog.
+     *
+     * The default list is in the config `service.catalog`. This is
+     * replaced at runtime by an API call.
+     *
+     * @param bool $runtime
+     *
+     * @return array
+     */
+    protected function getDefaultCatalog($runtime = false)
+    {
+        if ($runtime) {
+            $catalog = [];
+            $catalog_items = $this->api()->getClient()->getCatalog()->getData();
+            if (!empty($catalog_items)) {
```

Comment:
the `if` isn't needed - if it's always an array, the `foreach` won't mind if it's empty